src/main/java/frc/robot/subsystems/AutonSubsytem.java

```diff
@@ -206,10 +207,13 @@ private Command getBackupSequence(){
         //Backup sequence in case a trajectory fails to load
         return new SequentialCommandGroup(
             swerveSubsystem.followTrajectory(
-                "Up",
+                "Apriltag",
                 generateTrajectory(
-                    constructPoint(0, 0, 0, 90),
-                    constructPoint(0, 1, 0, 0)
+                    constructPoint(0, 0, 0, 0),
```

Comment:
0,0,0,0 is on the grid.  Should we have a different point for this?

Reply:
Just a placeholder trajectory